### PR TITLE
Fix/new notification

### PR DESCRIPTION
**Pull request by abdullahjankhan1**
## Description

Closes: #[PS-349](https://pylonstech.atlassian.net/browse/PS-349)

Implemented notifications for 50% and 100% NFT sold.
---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: big-dipper/imports/api/analytics/server/methods.js [hunk 1]
```diff
@@ -135,346 +137,346 @@ if (Meteor.isServer) {
...
     },
-    'Analytics.getSalesGraph': async function () {
-      const start = new Date()
-      const end = new Date()
-      start.setDate(start.getDate() - 7)
-      end.setDate(end.getDate() - 6)
+    "Analytics.getSalesGraph": async function () {
+      const start = new Date();
+      const end = new Date();
+      start.setDate(start.getDate() - 7);
```
Comment (mustafapylons) [inline big-dipper/imports/api/analytics/server/methods.js:321]: "7" should be in a constant.

File: big-dipper/imports/api/analytics/server/methods.js [hunk 2]
```diff
@@ -135,346 +137,346 @@ if (Meteor.isServer) {
...
+      end.setDate(end.getDate() - 6);
```
Comment (mustafapylons) [inline big-dipper/imports/api/analytics/server/methods.js:322]: "6" should be in a constant

**Review by mustafapylons — COMMENTED** [review]
few issues, otherwise code is good.

File: big-dipper/imports/api/notifications/server/methods.js [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
-const Success = 'Success'
-const BadRequest = 'Bad Request'
-const InvalidID = 'Invalid Notification ID'
-const AppCheckFailed = 'App Check Failed'
+const StatusOk = 200;
+const StatusInvalidInput = 400;
+const Success = "Success";
+const BadRequest = "Bad Request";
+const InvalidID = "Invalid Notification ID";
+const AppCheckFailed = "App Check Failed";
```
Comment (justyntemme) [inline big-dipper/imports/api/notifications/server/methods.js:16]: I thought we discussed there was no need to use appcheck in big dipper? How are we using appcheck?

Comment (abdullahjankhan1) [reply big-dipper/imports/api/notifications/server/methods.js:16]: Yes so we need the FCM token from our app so for that we have applied app-check verification before we set FCM. Plus it enhances a bit in terms of security.

File: big-dipper/imports/api/analytics/server/methods.js (hunk 1 above)
Comment (abdullahjankhan1) [reply big-dipper/imports/api/analytics/server/methods.js]: Resolved.
image

Updated to this.

**Review by faddat — APPROVED** [review]

File: big-dipper/imports/api/notifications/server/methods.js (hunk 3 above)
Comment (faddat) [reply big-dipper/imports/api/notifications/server/methods.js:16]: Is this resolved?

**Pull request closed without merging**